hardware/bluetooth/.gitignore

```diff
@@ -0,0 +1 @@
+/company-identifiers.html
```

Comment:
I have no idea what this is about.

Reply:
No, my script caches the webpage into this file.